packages/core/admin/scripts/build.js

```diff
@@ -2,7 +2,11 @@
 
 const path = require('path');
 const webpack = require('webpack');
+// eslint-disable-next-line node/no-extraneous-require
+const SpeedMeasurePlugin = require('speed-measure-webpack-plugin');
```

Comment:
I think the majority of our users won't need or use this package. What do you think about making this a peer dependency and lazy load it here, if `MESURE_BUILD_SPEED` is set (in case it's not installed just error)?

Reply:
I agree this is why I have added the dependency to the monorepo's package.json and not the admin one (thus the eslint-lint disable line).
Also there's no way to access this script in a generated app unless if it's run inside the node_module.

I wanted to add this script in order to be able to measure the speed without having to reinstall everything each time we want to make some tests.

WDYT?

Reply:
Ah, right. Didn't get that: I think it's a great idea. 👍🏼

Reply:
I think you should add it in the package devDependencies instead to avoid this eslint hack
In production user won't install the devDependencies